docs/intro/arch_overview/load_balancing.rst

```diff
@@ -32,6 +32,17 @@ weighted least request behavior is desired (generally if request durations are v
 length). We may add a true full scan weighted least request variant in the future to cover this use
 case.
 
+Ring hash
+^^^^^^^^^
+
+The ring/modulo hash load balancer implements consistent hashing to upstream hosts. The algorithm is
+based on mapping all hosts onto a circle such that any host set changes only affect 1/N requests.
```

Comment:
i think this is not 100% correct, specifically it's only single addition or removal of the host from host set will affect 1/N requests, but not `any` host set changes.
rephrase it a bit here?